docs/algorithms/kem/ml_kem.yml

```diff
@@ -0,0 +1,125 @@
+name: ML-KEM
+type: kem
+principal-submitters:
+- Peter Schwabe
+auxiliary-submitters:
+- Roberto Avanzi
+- Joppe Bos
+- Léo Ducas
+- Eike Kiltz
+- Tancrède Lepoint
+- Vadim Lyubashevsky
+- John M. Schanck
+- Gregor Seiler
+- Damien Stehlé
+crypto-assumption: Module LWE+R with base ring Z[x]/(3329, x^256+1)
+website: https://pq-crystals.org/kyber/ and https://csrc.nist.gov/pubs/fips/203/ipd
+nist-round: ipd
```

Comment:
This (string) designation causes downstream problems where "nist-round" is supposed to be an integer. Would it make sense to assign "ipd" a special new int value (sufficiently high to be above all possible future NIST rounds but also not so high such as to permit a designation for "second public draft", etc. until finalization)? Alternatively, I'll change all logic keeping algorithms in the NIST rounds together based on integer and move it to strings.

Reply:
> I'd rather move it to strings if possible

It is/done: https://github.com/open-quantum-safe/oqs-provider/pull/344